---
float32 degrees_cw